db/seeds.rb

```diff
@@ -2,9 +2,11 @@
 
 Rails.application.eager_load!
 
-Dir.glob(Rails.root.join('db/seeds/**/*.yml')).each do |f|
-  basename = Pathname.new(f).relative_path_from(Pathname.new(Rails.root.join('db/seeds'))).to_s
-  type = basename.gsub('.yml', '').singularize.classify.constantize
+RequestContext.community = Community.new(id: 1)
```

Comment:
This will need some more changes to make it work more widely, but I'm not entirely sure what that entails yet so we'll leave it for this PR.